mars/services/scheduling/supervisor/queueing.py

```diff
@@ -75,6 +75,8 @@ async def __post_create__(self):
         async def watch_bands():
             async for bands in self._cluster_api.watch_all_bands():
                 self._band_slot_nums = bands
+                if self._band_queues:
```

Comment:
Maybe we need to check if bands _really_ changed?

Reply:
As I understand, watch_all_bands() returns ready bands when some bands are marked as STOPPING or newly come up as READY. Isn't that correct? Codes highlighted here, I think, mean that there's no need to reassign subtasks when there's nothing in _band_queues.